lib/BoundaryClickWatcher/index.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Actually both these cases are easy to handle. We're running very old versions of the linters and apparently tons of discussions happened around this since then; both cases are fixable by adding `aria` roles when updating to the newer versions.

However... newer package versions also add more rules, so I now have again 100+ errors to fix 😂 

I think it's worth the hassle, though, so I'll update the packages and fix those.